protocol/x/clob/keeper/deleveraging.go

```diff
@@ -49,14 +50,40 @@ func (k Keeper) MaybeDeleverageSubaccount(
 		return new(big.Int), nil
 	}
 
-	telemetry.IncrCounter(
+	quantumsDeleveraged, err = k.MemClob.DeleverageSubaccount(ctx, subaccountId, perpetualId, deltaQuantums)
+
+	labels := []gometrics.Label{
+		metrics.GetLabelForIntValue(metrics.PerpetualId, int(perpetualId)),
+		metrics.GetLabelForBoolValue(metrics.IsLong, deltaQuantums.Sign() == -1),
+	}
+	telemetry.IncrCounterWithLabels(
+		[]string{types.ModuleName, metrics.DeleverageSubaccount},
 		1,
-		types.ModuleName,
-		metrics.PrepareCheckState,
-		metrics.DeleverageSubaccount,
+		append(
+			[]gometrics.Label{
+				metrics.GetLabelForBoolValue(metrics.Unfilled, quantumsDeleveraged.Sign() == 0),
+				metrics.GetLabelForBoolValue(
+					metrics.FullyFilled,
+					quantumsDeleveraged.CmpAbs(deltaQuantums) == 0,
+				),
+			},
+			labels...,
+		),
 	)
 
-	return k.MemClob.DeleverageSubaccount(ctx, subaccountId, perpetualId, deltaQuantums)
+	if quoteQuantums, err := k.perpetualsKeeper.GetNetNotional(
```

Comment:
Should we log out `err` if it's non-nil?

Reply:
ideally we would panic since this is not really supposed to happen. but don't want to panic for telemetry. 

if this returns an error, I am pretty sure we will have logs somewhere, so leaning towards not logging this here